rust+wasm/.github/workflows/tests_and_checks.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           - stable
           - nightly
           # minimum version
-          - 1.67
+          - "1.75"
```

Comment:
do we need the quotes here?

Reply:
We don't need them for 1.75, however, we would need them for `1.70`, or `1.80` as the yaml parser will interpret those as numbers and shorten them to `1.7` and `1.8` respectively. I saw this happen when testing and one of the libs needed `1.70` and the action installed `1.7` instead.